tests/e2e/cucumber/features/integrations/link.feature

```diff
@@ -1,6 +1,7 @@
 Feature: link
 
-  Scenario: public link
+  @skipOnOC10
```

Comment:
we've discussed that today and decided to not introduce test tags for now at all.
The idea was to have 3 types of feature files:

* link.feature
* link.ocis.feature
* link.oc10.feature

then wa're able to decide on the the cli level which tests to run by using an glob.

* ocis only -> any-path/*{!.oc10}.feature
* oc10 only -> any-path/*{!.ocis}.feature

please have a look and try it out if it makes any sense for you.

Reply:
This is implemented now in latest master